libs/net/dal/Migrations/1.0.5/Down/PreDown/00-Tag.sql

```diff
@@ -0,0 +1,21 @@
+DO $$
+BEGIN
+
+CREATE TEMP TABLE _tag AS
+SELECT "id", "code"
+FROM public.tag;
+
+ALTER TABLE public.content_tag DROP CONSTRAINT "FK_content_tag_tag_tag_id";
+
+ALTER TABLE public.tag ALTER COLUMN "id" DROP IDENTITY IF EXISTS;
+
+ALTER TABLE public.content_tag ALTER COLUMN "tag_id" SET DATA TYPE varchar USING "tag_id"::varchar;
+ALTER TABLE public.tag ALTER COLUMN "id" SET DATA TYPE varchar USING "id"::varchar;
+
+UPDATE public.tag
+SET "id" = _tag.code
+FROM _tag
+WHERE _tag.code = public.tag.code;
+
+ALTER TABLE public.content_tag ADD CONSTRAINT "FK_content_tag_tag_tag_id" FOREIGN KEY ("tag_id") REFERENCES public.tag ("id") ON DELETE CASCADE;
```

Comment:
This should fail because you've converted the `tag_id` to a `varchar` and it will have the old foreign key number values.  Which will not align with the new foreign key to the tag table that is now a `varchar`.